juniper/src/lib.rs

```diff
@@ -293,7 +296,10 @@ where
 
         let errors = ctx.into_errors();
         if !errors.is_empty() {
-            return Err(GraphQLError::ValidationError(errors));
+            let gql_error = GraphQLError::ValidationError(errors);
+            __juniper_trace_error!("GraphQLError: {:?}", gql_error);
```

Comment:
Should this traces be `error` or would they sit better as `debug`?

Reply:
Let's make them `trace`, which is the lowest.